src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator {
```

Comment:
Consider making `Figure` an abstract class if it is intended to serve as a base class for other specific figure types like `Circle`. This would prevent direct instantiation of `Figure` and emphasize that it is meant to be extended.